x/staging/posts/simulation/utils.go

```diff
@@ -89,7 +87,7 @@ func RandomPostData(r *rand.Rand, accs []simtypes.Account) PostData {
 	post := types.NewPost(
 		"",
 		RandomPostID(r),
-		RandomMessage(r)+RandomHashtag(r),
```

Comment:
Why has this been deleted?

Reply:
@RiccardoM  Desmos does not filter post with hashtag anymore, so I removed all hashtag related things.

Reply:
@dadamu Where they only used inside filtering?

Reply:
@RiccardoM Filtering only happens when querying.

Reply:
Ok then we can remove them :+1: